runtime/queries/r/highlights.scm

```diff
@@ -0,0 +1,128 @@
+; highlights.scm
+
+
+; Literals
+
+(integer) @number
+
+(float) @float
+
+(complex) @number
```

Comment:
`@constant.numeric.integer`, etc.

Queries need to follow these scope rules https://docs.helix-editor.com/themes.html#syntax-highlighting